app/renderer/components/navigation/navigationBar.js

```diff
@@ -292,13 +292,14 @@ class NavigationBar extends React.Component {
         titleMode={this.props.titleMode}
         onStop={this.onStop}
         menubarVisible={this.props.menubarVisible}
-        />
+      />
       {
-        isSourceAboutUrl(this.props.location)
+        isSourceAboutUrl(this.props.location) && !this.props.titleMode
```

Comment:
This causes a bug, which will be fixed with https://github.com/brave/browser-laptop/pull/9299/commits/3253152543f225e076254765bf16be5df9f36565